apps/faqs/admin.py

```diff
@@ -1,22 +1,33 @@
 from cms.admin import PageBaseAdmin, SearchMetaBaseAdmin
+from suit.admin import SortableModelAdmin
 from django.contrib import admin
 
-from .models import Category, Faq
+from .models import Category, Faq, Faqs
 
 
 @admin.register(Faq)
-class FaqAdmin(SearchMetaBaseAdmin):
-    prepopulated_fields = {'url_title': ('question',)}
+class FaqAdmin(SearchMetaBaseAdmin, SortableModelAdmin):
+    prepopulated_fields = {'slug': ('question',)}
```

Comment:
Put an empty line under this - class attributes should usually have an empty line underneath. (It was like this before you got to it, but we may as well do it now.)